### DISCUSSION TITLE
Problem to get the content watch and the previous

### DISCUSSION THREAD

**Discussion by tomasmalio** [chamilo/chamilo-lms#5175] [General]
I have a problem that I am wanting to obtain the content that I am viewing and from which I come in a scorm that I have inside my Chamilo.

I can't get it, I have modified the lp_controller but I don't always get the same thing.

Could someone help me if this happens to them?

**Comment by ywarnier** [comment]
Hi @tomasmalio 
I'm not sure I understand the problem. You want to "obtain" how? You want to download it? View it in the browser? Your SCORM content doesn't work? We need more info to help.
